src/Files.App/Utils/Logger/SentryLogger.cs

```diff
@@ -23,7 +23,12 @@ public bool IsEnabled(LogLevel logLevel)
 		public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
 		{
 			// Unhandled exceptions are captured in AppLifecycleHelper.HandleAppUnhandledException
-			if (exception is null || exception.Data[Mechanism.HandledKey] is false)
+			if
+			(
+				exception is null ||
+				exception.Data[Mechanism.HandledKey] is false ||
+				logLevel <= LogLevel.Information
+			)
```

Comment:
```suggestion
			if(exception is null ||
				exception.Data[Mechanism.HandledKey] is false ||
				logLevel <= LogLevel.Information)
```

Reply:
Isn't it easier to read if all the conditions are aligned?

Reply:
Worsen for me. I don't get the reason why brackets have its own line break.